lib/fizzy/parser_expr.cpp

```diff
@@ -120,7 +121,7 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, const Mod
     // The stack of control frames allowing to distinguish between block/if/else and label
     // instructions as defined in Wasm Validation Algorithm.
     // For a block/if/else instruction the value is the block/if/else's immediate offset.
-    std::stack<ControlFrame> control_stack;
+    Stack<ControlFrame> control_stack;
```

Comment:
Does this makes any measurable difference in speed?

Reply:
This uses `std::vector` instead of `std::deque` now, but it seems make no difference (see below).